<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PadTool</class>
 <widget class="QWidget" name="PadTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>70</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>24</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>24</number>
         </attribute>
         <row>
          <property name="text">
           <string>Name</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Type</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Width</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Hole</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Diameter</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Via</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Bus</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Orientation</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Count</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Spacing</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Property</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Name</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>Type</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>Width</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>Hole</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>Diameter</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>Via</string>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>Bus</string>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string>Orientation</string>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string>Count</string>
          </property>
         </item>
         <item row="9" column="0">
          <property name="text">
           <string>Spacing</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QCheckBox" name="holeCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>44</width>
         <height>17</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="typeComboBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>128</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox { background-color: white; } </string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Round</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/roundpad.png</normaloff>:/pictures/roundpad.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Square</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/squarepad.png</normaloff>:/pictures/squarepad.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Round Square</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/roundsquarepad.png</normaloff>:/pictures/roundsquarepad.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polygon Square</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/polysquarepad.png</normaloff>:/pictures/polysquarepad.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ellipse</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/ellipsepad.png</normaloff>:/pictures/ellipsepad.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Round Rectangle</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/roundrectpad.png</normaloff>:/pictures/roundrectpad.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rectangle</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/rectpad.png</normaloff>:/pictures/rectpad.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polygon</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/polyrectpad.png</normaloff>:/pictures/polyrectpad.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hole</string>
        </property>
        <property name="icon">
         <iconset resource="../../mytypon.qrc">
          <normaloff>:/pictures/hole.png</normaloff>:/pictures/hole.png</iconset>
        </property>
       </item>
      </widget>
      <widget class="MYUnitsSpinBox" name="widthSpinBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>53</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="busCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="busComboBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>74</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox { background-color: white; } </string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Horizontal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Vertical</string>
        </property>
       </item>
      </widget>
      <widget class="QSpinBox" name="busSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>190</y>
         <width>37</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
      </widget>
      <widget class="MYUnitsSpinBox" name="busSpacingSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>220</y>
         <width>53</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="MYUnitsSpinBox" name="holeSpinBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>100</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="viaCheckBox">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>130</y>
         <width>37</width>
         <height>17</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MYUnitsSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>utils/myunitsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../mytypon.qrc"/>
 </resources>
 <connections/>
</ui>
